reactioncommerce/reaction
# feat: authentication plugin; drop support for meteor-login-token header

Resolves #5998   
Impact: **major**  
Type: **feature**

NOTE: Do not merge until https://github.com/reactioncommerce/reaction-admin/pull/171 has been merged!

## Changes
- "Authorization" header handling remains the same but is moved from `accounts` plugin to a new `authentication` plugin.
- "meteor-login-token" header is no longer supported. This breaks all releases of Reaction Admin prior to [this PR](https://github.com/reactioncommerce/reaction-admin/pull/171)

## Breaking changes
"meteor-login-token" header is no longer supported. This breaks all releases of Reaction Admin prior to [this PR](https://github.com/reactioncommerce/reaction-admin/pull/171)

## Testing
- Verify authenticating a GraphQL request with "Authorization" header works as previously
- Verify authenticating a GraphQL request with "meteor-login-token" header no longer works
